modules/twinklewarn.js

```diff
@@ -168,34 +168,44 @@ Twinkle.warn.messages = {
 	level1: {
 		"Common warnings": {
 			"uw-vandalism1": {
-				label: "Vandalism",
-				summary: "General note: Unconstructive editing"
+				label: "Blatant vandalism",
+				summary: "General note: Blatant vandalism"
```

Comment:
I think this is a bit strong for [uw-vandalism1](https://en.wikipedia.org/wiki/Template:Uw-vandalism1); I'd prefer "Unconstructive editing" for this level, but it seems okay for later levels.

Reply:
I took the description straight from WP:UTM, but I agree that this wording is too strong for uw-vandalism1.